install_snapshot.sh

```diff
@@ -10,13 +10,12 @@ archive_file="snapshot_core_all_adapters_${suffix}.zip"
 requirements_file="snapshot_requirements_${suffix}.txt"
 echo $suffix
 export DBT_PSYCOPG2_NAME=psycopg2
-curl --fail --retry 5 --retry-all-errors -OL "${github_url}/${archive_file}"
```

Comment:
Should be fine to delete `--retry-all-errors`, just a note that originally I had included this since I was running into some eventual consistency latency with github

Reply:
I'm not sure why I removed it so I'll add it back.